vars/notification.groovy

```diff
@@ -13,7 +13,7 @@ def send_notification(subject, mail_body, version){
 
 def send_notification_infra(subject, mail_body, version){
   mail subject: subject, \
-    to: "rhev-qe-infra", \
+    to: "rhv-qe-devops", \
```

Comment:
please define constant variable for it as whole email rhv-qe-devops@redhat.com, we use it on multiple places so see my previous comment about it